x-pack/plugins/data_visualizer/public/application/file_data_visualizer/components/file_contents/file_contents.tsx

```diff
@@ -132,13 +133,23 @@ export const FileContents: FC<Props> = ({
 
       <EuiSpacer size="s" />
 
-      <FormattedMessage
-        id="xpack.dataVisualizer.file.fileContents.firstLinesDescription"
-        defaultMessage="First {numberOfLines, plural, zero {# line} one {# line} other {# lines}}"
-        values={{
-          numberOfLines: showHighlights ? LINE_LIMIT : numberOfLines,
-        }}
-      />
+      {format === FILE_FORMATS.TIKA ? (
+        <FormattedMessage
+          id="xpack.dataVisualizer.file.fileContents.characterCount"
+          defaultMessage="First {numberOfChars} characters"
```

Comment:
Do you know the total number of chars shown in the file contents preview section? Just wondering if this could be hidden if we know that the file contains less than 100000 chars?

Reply:
The only way to get actual number of chars is by parsing the whole file, but this limit is here to avoid having to do that just for this content preview.
The text could be changed to `Limited to the first 100000 characters` or something like that?

Reply:
Updated in https://github.com/elastic/kibana/pull/197333/commits/7888ea898b64743361f0a26aa8c8b8b239f348c3

Reply:
Updated again so it is now `Preview limited to the first 100000 characters`
So not to give the impression that the whole feature is limited to this char count.